src/vmm/src/linux/vstate.rs

```diff
@@ -1129,6 +1131,14 @@ impl Vcpu {
                     error!("Received KVM_EXIT_INTERNAL_ERROR signal");
                     Err(Error::VcpuUnhandledKvmExit)
                 }
+                VcpuExit::SystemEvent(event, _reason) => {
```

Comment:
Should this be tagged behind `#[cfg(target_arch = "aarch64")]`?

Reply:
On x86_64 we don't hit that one right now, because we shut down with a triple fault, but it's valid a could eventually happen.